Gemfile.lock

```diff
@@ -369,4 +368,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.3.9
+   2.3.8
```

Comment:
Nitpick.

I'd be good to only _upgrade_ the expected Bundler version. What do you think? 

I've never looked into it, but maybe there's a way to actually lock it. `bundle install` warns us if the version is below the required one, but I wonder if we can stop it from even moving on with the process. ...Adding it to the "things I'll look into when I'll have some time" list 🙃

Reply:
Wow I'm actually surprised that the downgrade was allowed to happen here, good catch; I'd have expected my run of `bundle update fastlane` would have warned me about it at the very least 🤔 (or maybe it did and I was not attentive enough with the console output?)

I agree about only going up in versions; I'll upgrade my local Bundler version and re-run it to fix that; thanks!

Reply:
Ah shoot just realized auto merge already kicked in 🙃
Not sure it's worth doing a PR _just_ for that minor version bump off Bundler in lock file now that this one has already landed then … 🤪